docs/asciidoc/advancedFeatures.adoc

```diff
@@ -838,8 +849,8 @@ the following example shows:
 ----
 String key = "message";
 Mono<String> r = Mono.just("Hello")
-                .flatMap( s -> Mono.subscriberContext() //<2>
-                                   .map( ctx -> s + " " + ctx.get(key))) //<3>
+                .flatMap( s -> Mono.deferWithContext(Mono::just) //<2>
```

Comment:
Since this section is "simple `Context` examples", I'd suggest to use more... errr... "modern" way of accessing `Context` :D 

This example can at least be rewritten as:
```java
.flatMap(s -> Mono.deferWithContext(ctx -> Mono.just(s + " " + ctx.get(key))));
```

Reply:
ok